extensions/openwsman/processors/SourceInitiatedSubscriptionListener.cpp

```diff
@@ -887,7 +887,7 @@ void SourceInitiatedSubscriptionListener::onSchedule(const std::shared_ptr<core:
 }
 
 void SourceInitiatedSubscriptionListener::notifyStop() {
-  server_.release();
+  server_.release();  // NOLINT(bugprone-unused-return-value)
```

Comment:
Isn't this a real bug? I think it should be reset, not release.

Reply:
I think this is, good catch, updated in 3b4102f32349a842f90c11b08189d9e79411eca3